src/state/bridge/hooks.ts

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios'
+import { useQuery } from 'react-query'
+import { DIRECTUS_GRAPHQL_URL } from 'src/constants'
+
+export interface SubCategories {
+    id: number
+    title: string
+    content: string
+    subcategory: string
+  }
+  
+export function useSubBridgeCategories(subCategory: string) {
```

Comment:
this and `useGetQuestions` in `src/state/airdrop/hooks.ts` both are almost same, with just some variation. please make it common and reusable so we dont need to repeat same code